### DISCUSSION TITLE
change color of screen in COSMOS

### DISCUSSION THREAD

**Discussion by RockZombie4** [CosmosOS/Cosmos#2251] [Q&A]
Recently, I got COSMOS to work. 
I decided to change the color of the screen by using this:
```
                Console.Clear();
                // Fill background
                Console.BackgroundColor = ConsoleColor.Cyan;
```

But it didn't work, and instead, I got this:
![image](https://user-images.githubusercontent.com/100057184/170796447-07dd86b1-9f0c-4b6f-b0b5-79b328725776.png)

**Comment by ghost** [comment]
Try this:
```cs
Console.BackgroundColor = ConsoleColor.Cyan;
Console.Clear();
```

**Comment by LoxyMono** [comment]
The background color of the text is the only thing changing, console.clear will not clear it with the set color

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/98218366/170814812-69f9f5f3-e0cb-4b94-ae17-f3874eef1945.png)

**Comment by ghost** [comment]
It works

**Reply by LoxyMono** [reply]
Oh, guess I missed that feature?
